### PR TITLE
Task completed: GP-95 HP Collectable

### PR DESCRIPTION
**Pull request by frnzee**
Requires https://github.com/LighteatersStudio/TopDownShooter/pull/87 to complete task.

- added collectable item installer prefab
- added first aid kit models from asset store
- added first aid kit spawner prefab
- added first aid prefabs
- added collectable items rotation script

### CODE REVIEW

File: Assets/Scripts/Gameplay/Character/CharacterStats.cs [hunk 1]
```diff
@@ -30,5 +29,12 @@ public void ApplyDamage(float damage)
 
             HealthChanged?.Invoke();
         }
+
+        public void ApplyHp(float healthUp)
```
Comment (ega-ega) [inline Assets/Scripts/Gameplay/Character/CharacterStats.cs]: rename to Heal

File: Assets/Scripts/Gameplay/Collectables/RotateCollectables.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace Gameplay.Collectables
+{
+    public class RotateCollectables : MonoBehaviour
+    {
+        [SerializeField] private float _rotationSpeed = 100f;
+
+        private void Update()
+        {
+            transform.Rotate(Vector3.up, _rotationSpeed * Time.deltaTime);
```
Comment (ega-ega) [inline Assets/Scripts/Gameplay/Collectables/RotateCollectables.cs]: Pls use DoTween

**Review by ega-ega — APPROVED** [review]

File: Assets/Scripts/Gameplay/Character/CharacterStats.cs (hunk 1 above)
Comment (frnzee) [reply Assets/Scripts/Gameplay/Character/CharacterStats.cs]: done

File: Assets/Scripts/Gameplay/Collectables/RotateCollectables.cs (hunk 2 above)
Comment (frnzee) [reply Assets/Scripts/Gameplay/Collectables/RotateCollectables.cs]: done

**Review by ega-ega — APPROVED** [review]

**Pull request merged**